css-two-column-layout/index.html

```diff
@@ -0,0 +1,88 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>CSS Challenge - Two Column Layout</title>
+    <link rel="stylesheet" href="fonts.css" />
+    <link rel="stylesheet" href="reset.css" />
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <div class="container">
+      <div class="row">
+        <div class="column-full">
+          <h1>Scale, Speed, Agility</h1>
+          <div class="head">
+            We've dedicated thousands of man hours to making healthcare
+            institutions in the Helium network run 10x more efficiently and
+            deliver an unparalleled quality of care.
+          </div>
+        </div>
+      </div>
+      <div class="row">
+        <div class="column-eigth">
```

Comment:
I would recommend wrapping you `column-eighth` and `column-two-eighth` elements with a div that is `column-half` since each of the sections is half of a total row. That will also allow you more control in adjusting the images to the proper position.